cognee/modules/cognify/graph/add_classification_nodes.py

```diff
@@ -2,11 +2,11 @@
 from cognee.infrastructure.databases.graph.get_graph_client import get_graph_client, GraphDBType
 
 
-async def add_classification_nodes(document_id, classification_data):
-    graph_client = get_graph_client(GraphDBType.NETWORKX)
-
-
-    await graph_client.load_graph_from_file()
+async def add_classification_nodes(graph_client, document_id, classification_data):
```

Comment:
I would call it just `graph` instead of `graph_client`. Function doesn't need to know what it is, just how to use it.